microsoftgraph/microsoft-graph-explorer-v4
# Fix: Request headers textfield bug

## Overview

Fixes #1123 
Where a user couldn't erase the first character once typed in on the request header text fields

## Testing Instructions

* Navigate to the Request Header tab
* Type in a Key and Value...
* Erase the Key and Value
* Observe all characters are erased
